proto/valorem/trade/v1/auth.proto

```diff
@@ -28,4 +34,10 @@ message NonceText {
 /** Verify request message containing an JSON encoded string of the `SignedMessage` structure (src/auth). */
 message VerifyText {
   string body = 1;
+}
+
+/** The object representing a valid/authenticated sid */
+message SiweSession {
+  H160 address = 1;
+  H256 chain_id = 2;
```

Comment:
Does this need to be an h256? Seems like a u64 would do just fine.

Reply:
i copied your spec in the ticket, but yes agree, never bigger than u64

Reply:
will update